luigi/scheduler.py

```diff
@@ -224,10 +226,18 @@ def __init__(self, task_id, status, deps, resources=None, priority=0, family='',
         self.status_message = status_message
         self.scheduler_disable_time = None
         self.runnable = False
+        self.batchable = False
+        self.batch_id = None
 
     def __repr__(self):
         return "Task(%r)" % vars(self)
 
+    def is_batchable(self):
```

Comment:
Right, added a TODO comment for a year from now since it'll probably be a while before this makes it to most end users.